test/test-regexes.js

```diff
@@ -45,28 +67,10 @@ function testToken(feature, matches, misses) {
   );
 }
 
-/** @type {TestCase[]} */
-const tests = [
-  {
-    features: ['css.properties.color.alpha_hexadecimal_notation'],
-    matches: ['#003399ff', '#0af9'],
-    misses: ['#00aaff', '#0af', 'green', '#greenish'],
-  },
-  {
-    features: ['css.properties.transform-origin.three_value_syntax'],
-    matches: [
-      '2px 30% 10px', // length, percentage, length
-      'right bottom -2cm', // two keywords and length
-      'calc(50px - 25%) 2px 1px', // lengths with calc
-    ],
-    misses: [
-      'center', // one value syntax
-      'left 5px', // two value syntax
-      'left calc(10px - 50%)', // two value syntax with calc
-    ],
-  },
-];
+const testRegexes = () => {
+  tests.forEach(({ features, matches, misses }) => {
+    features.forEach((feature) => testToken(lookup(feature), matches, misses));
+  });
+};
 
-tests.forEach(({ features, matches, misses }) => {
-  features.forEach((feature) => testToken(lookup(feature), matches, misses));
-});
+module.exports = testRegexes;
```

Comment:
This script isn't supposed to be run on its own, but rather as a Mocha unittest.  It was never actually hooked up though, and I'm not too worried about hooking it up now since it's going to be removed in #15781!